source/.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
```

Comment:
Don't we have this file at the root path?

Reply:
it's the root of whatever folder you open your editor in